packages/testnet-faucets/src/ts/id_management.ts

```diff
@@ -18,4 +20,16 @@ export const idManagement = {
         const rawTx = `0x${tx.serialize().toString('hex')}`;
         callback(null, rawTx);
     },
+    signMessage(message: object, callback: Callback) {
+        const data = _.get(message, 'data');
+        if (_.isUndefined(data)) {
+            callback(new Error('No data to sign'), null);
```

Comment:
Let's make this `NO_DATA_TO_SIGN` so it's easier for the caller to check for it.